src/main/cliapp/src/components/EllipsisTableCell.js

```diff
@@ -2,8 +2,7 @@ import React from 'react';
 
 export const EllipsisTableCell = ({ children, otherClasses = '' }) => {
 	return (
-		<div className={`overflow-hidden text-overflow-ellipsis ${otherClasses}`} >
-			{children}
+		<div className={`overflow-hidden text-overflow-ellipsis ${otherClasses}`} dangerouslySetInnerHTML={{__html: children}}>
```

Comment:
@kthorat-prog I think this might cause us problems in the future doing at this level.